.github/workflows/stale.yml

```diff
@@ -13,8 +13,8 @@ jobs:
         with:
           stale-issue-label: "stale"
           stale-issue-message: 'This issue has been automatically marked as stale because it has been open for 7 days without activity. It will be closed if no further activity occurs. If this is still an issue, just leave a comment or remove the "stale" label. 🙂'
```

Comment:
```suggestion
          stale-issue-message: 'This issue has been automatically marked as stale because it has been open for 14 days without activity. It will be closed if no further activity occurs within the next 14 days. If this is still an issue, just leave a comment or remove the "stale" label. 🙂'
```